activiti-cloud-starter-runtime-bundle/src/test/java/org/activiti/cloud/starter/tests/services/audit/AuditProducerIT.java

```diff
@@ -556,7 +558,67 @@ public void shouldEmitEventsForTaskAddDeleteGroupCandidates() {
             List<CloudRuntimeEvent<?, ?>> receivedEvents = streamHandler.getLatestReceivedEvents();
         });
     }
+    
+    
+    @Test
+    public void testTwoSubProcesses() {
+        //given
+        ResponseEntity<CloudProcessInstance> processInstance = processInstanceRestTemplate.startProcess(processDefinitionIds.get("callTwoSubProcesses"));
+        
+        String processInstanceId = processInstance.getBody().getId();
+        
+        // when
+        List<String> subprocessIds = runtimeService.createProcessInstanceQuery()
+                                                   .superProcessInstanceId(processInstanceId)
+                                                   .list()
+                                                   .stream()
+                                                   .map(it -> it.getProcessInstanceId())
+                                                   .collect(Collectors.toList());
+        // then
+        assertThat(subprocessIds).hasSize(2);
+        
+        String subProcessId1 = subprocessIds.get(0);
+        String subProcessId2 = subprocessIds.get(1);
+        
+        await().untilAsserted(() -> {
+            assertThat(streamHandler.getReceivedHeaders()).containsKeys(ALL_REQUIRED_HEADERS);
+
+            assertThat(streamHandler.getLatestReceivedEvents())
+                    .extracting("eventType","processInstanceId","parentProcessInstanceId", "processDefinitionKey")
```

Comment:
Literal strings can be replaced by method references:
`.extracting(CloudRuntimeEvent::getEventType, CloudRuntimeEvent::getProcessInstanceId,CloudRuntimeEvent::getParentProcessInstanceId, CloudRuntimeEvent::getProcessDefinitionKey)`

Reply:
@erdemedeiros Good suggestion. I like to use method references for extracting attributes